internal/bminventory/inventory_test.go

```diff
@@ -5665,6 +5665,48 @@ var _ = Describe("cluster", func() {
 			})
 		})
 
+		It("Should handle not found correctly on fetch of UI specific settings for cluster", func() {
```

Comment:
nit, but a better name for this would explain the condition when this situation happens rather than "handles not found correctly"

So something like:
```
It("returns not found when the cluster doesn't exist", func() {
```

Also I'd group all these function that are testing `V2GetClusterUISettings` in a `Describe` block.